### PR TITLE
feat: added netlify-cli package and script to generate a storybook link

### PR DESCRIPTION
**Pull request by direshp**
### JIRA Ticket

Jira:  [BB-8497](https://hello-haptik.atlassian.net/browse/BB-8497)

### Changelog
- added netlify-cli package and script to generate a storybook link

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -128,6 +129,7 @@
     "style-loader": "^1.0.0",
     "styled-components": "^5.3.0",
     "styled-system": "^5.1.5",
+    "typescript": "^3.9.7",
```
Comment (sidMulay) [inline package.json:132]: Why was typescript upgrade needed?

**Review by sidMulay — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (direshp) [reply package.json:132]: There was no typescript update done and the version is the same as well but its showing there is some change not sure why.

Comment (direshp) [reply package.json:132]: I think only the line number where the package was mentioned is changed.

**Review by ChiragJain-Haptik — APPROVED** [review]

**Review by sidMulay — APPROVED** [review]

**Pull request merged**